resource_booking_combination_auto_create/models/resource_booking.py

```diff
@@ -0,0 +1,36 @@
+# SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from odoo import models
+
+
+class ResourceBooking(models.Model):
+    _inherit = "resource.booking"
+    # override constraint to allow to save resource bookings without
+    # combinations in pending state.
+    _sql_constraints = [
+        (
+            "combination_required_if_event",
+            "check(meeting_id is null or state = 'pending' or combination_id is not null)",
+            "Missing resource booking combination.",
+        ),
+    ]
```

Comment:
@pedrobaeza what do you think? this change is to allow to save a booking with a date but no resource combination, and leave it in `pending` state. currently this is not allowed by `resource_booking`, because as soon as a date is set, it considers the booking to be in `scheduled` state, and that requires a resource combination.